lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -218,7 +218,7 @@ def add_txn_id(xml, authorization)
       end
 
       def add_network_token(xml, payment_method)
-        payment = payment_method.payment_data&.gsub('=>', ':')
+        payment = payment_method.payment_data.to_s&.gsub('=>', ':')
```

Comment:
Yes it is because as part of the worldpay change we are standardizing sending payment data as an object instead of a string in some places and an object in others. So this change allows elevon to continue to work with the new design.